tests/phpunit/test-wc-stripe-helper.php

```diff
@@ -179,7 +179,7 @@ public function test_get_legacy_enabled_payment_method_ids() {
 
 		$result = WC_Stripe_Helper::get_legacy_enabled_payment_method_ids();
 		// In legacy mode (when UPE is disabled), Stripe refers to Card as payment method.
-		$this->assertEquals( [ WC_Stripe_Payment_Methods::CARD, WC_Stripe_Payment_Methods::EPS, WC_Stripe_Payment_Methods::GIROPAY, WC_Stripe_Payment_Methods::P24 ], $result );
```

Comment:
Why is `card` no longer part of the default enabled payment methods?

Reply:
It's not present in the `develop` branch... I guess it was needed for the old enable_upe/disable_upe changes